deployments/cluster/educator.nix

```diff
@@ -82,14 +81,37 @@ in
       };
     };
 
-    args = let
-      cat = path: ''"$(cat "${path}")"'';
-    in {
+    args = {
       inherit config-key;
       bind = address "*";
       peer = map (node: address node.config.networking.privateIPv4)
         (attrValues (filterAttrs (name2: node: name != name2 && hasWitnessTag node) nodes));
     };
+
+    after = [ "postgresql.service" ];
+    serviceConfig = {
+      ExecStartPre =
```

Comment:
You will need `PermissionsStartOnly = true` in order to run as root so it can run sudo.

Reply:
Nope, I use prefixes, and I think that it’s better as it is more explicit.